hooks/cascading-scans/hook.ts

```diff
@@ -25,8 +27,9 @@ interface HandleArgs {
 export async function handle({ scan, getFindings }: HandleArgs) {
   const findings = await getFindings();
   const cascadingRules = await getCascadingRules(scan);
+  const cascadedRule = await getCascadedRule(scan);
```

Comment:
This var should probably have a different name to make it clearer what the distinction to the rules in `cascadingRules` is. Maybe something explicit like `cascadedRuleUsedForCurrentScan`.